packages/ui/src/theme/createTheme/resolveObject.ts

```diff
@@ -0,0 +1,148 @@
+// Internal Style Dictionary methods
+// copied from amzn/style-dictionary with the owner's permission
```

Comment:
Think we can remove this comment since over time the code will likely deviate (which it already is in this PR)

Reply:
Removed